api/src/services/tickets/tickets.js

```diff
@@ -10,8 +19,27 @@ export const ticket = ({ id }) => {
   })
 }
 
-export const createTicket = ({ input }) => {
+export const lastTicket = () => {
+  return db.ticket.findFirst({
+    orderBy: [{
+      number: 'desc'
+    }],
+  })
+}
+
+export const createTicket = async ({ input }) => {
+  var lastTicket = await db.ticket.findFirst({orderBy: [{number: 'desc'}],});
+  input.number = (parseInt(lastTicket.number,10)+1).toString()
+  var modifiedInput;
+  rulesArr.forEach((rule)=>{
+    console.log('running', rule, input)
+    modifiedInput = rule.command(input);
+    console.log('ran', rule, input)
+  })
+  //var modifiedInput = await runRules(input, 'tickets');
+  console.log(`modifiedInput ${JSON.stringify(modifiedInput)}`)
```

Comment:
remove logging or put it as debug

Reply:
Removed added backlog item to add debugging class